search/processor.hpp

```diff
@@ -140,16 +115,13 @@ class Processor : public my::Cancellable
   set<uint32_t> m_preferredTypes;
 
   m2::RectD m_viewport;
-  m2::PointD m_pivot;
   m2::PointD m_position;
-  double m_minDistanceOnMapBetweenResults;
-  Mode m_mode;
-  bool m_suggestsEnabled;
-  shared_ptr<hotels_filter::Rule> m_hotelsFilter;
-  bool m_cianMode = false;
-  SearchParams::OnResults m_onResults;
-
-  bool m_viewportSearch;
+  bool m_needAddress = true;
+  bool m_needHighlight = true;
```

Comment:
here too, please
and in the commit message

Reply:
These fields must be removed.